src/resmom/Makefile.am

```diff
@@ -96,7 +98,9 @@ pbs_mom_SOURCES = \
 	rm_dep.h \
 	stage_func.c \
 	start_exec.c \
-	vnode_storage.c
+	vnode_storage.c \
+	renew.c \
+	renew.h
```

Comment:
the name renew is too generic, i would have named gss_kerb5_renew_creds.c/h

Reply:
GSS takes no place here. I would go with renew_creds.c/h. The functions here are suitable for both general renewing credentials and for renewing krb5 credentials, so I suggest going with quite a general name.

Reply:
i like renew_creds.c/h

Reply:
hey @vchlum - did we not agree to change the name to renew_creds.c/h. renew is too generic for me.

Reply:
Yes, I totally agree with renew_creds.c/h. The change will come in the next commit. I wanted to wait for more comments before the next commit. I will push the commit soon.

In order to keep the order and do not forget any comment, I will use 'thumb up' when I do the change on local.